schedule/virt_autotest/kubevirt-tests.yaml

```diff
@@ -0,0 +1,38 @@
+name:           kubevirt-tests
+description:    >
+    Maintainer: nan.zhang@suse.com
+    Kubevirt server & agent node installation and test modules
+vars:
+    SCC_ADDONS: legacy,containers,sdk
```

Comment:
Does 'containers' in your SCC_ADDONS work? I saw the definition in lib/: 
```
    sdk => 'Development-Tools',
    desktop => 'Desktop-Applications',
    legacy => 'Legacy',
    script => 'Web-Scripting',
    serverapp => 'Server-Applications',
    contm => 'Containers',
```
'contm' is supposed to be a common keyword to be used to install 'Containers' addon.

Reply:
Corrected. Thanks